src/parser/jobs/drk/modules/GCDs.js

```diff
@@ -56,13 +56,21 @@ export default class GCDs extends Module {
 	_lastComboGCDTimeStamp = undefined
 	_GCDChainDrops = []
 
+	// Recommendation severities
+	_severityDroppedGCDCombo = {
```

Comment:
Normally peeps seem to use a const for this since it's up top and easy to find, plus don't need to do `this._reeeee` to reference it.

Reply:
Ah, I aped the way RDM\Dualcast did it.  I can adjust to a module-global const.